skxray/core/fitting/lineshapes.py

```diff
@@ -385,3 +391,100 @@ def compton(x, compton_amplitude, coherent_sct_energy,
     counts += value
 
     return counts
+
+
+def gamma_dist(bin_values, K, M):
```

Comment:
wikipedia says κ and θ  If we drop 2.7 we could use those as the variables!

https://en.wikipedia.org/wiki/Gamma_distribution